### DISCUSSION TITLE
Problem

### DISCUSSION THREAD

**Discussion by Philipock** [the-marenga/sf-scrapbook-helper#87] [General]
The helper takes much longer then bevor too fetch players. used to go very fast but now it stops every 200 players for some minutes. Is this intended?

**Comment by the-marenga** [comment]
Please read the info box in the release notes, or the main page:

> [!IMPORTANT]  
> S&F added rate-limiting (200reqs/min), so the ability to manually crawl an entire server is limited for the foreseeable future. I will work on updating the pre-crawled player lists, so that this will largely not be required

**Reply by Tuscad** [reply]
In connection with the above blocking.

Fetching a HoF that has 200,000 players takes a very long time these days.  If I fetch this database from morning to evening, are the results from this database updated on an ongoing basis? Or  the results available in the morning become outdated? 

I use this program to look for very rare items and I don't know if ZHOF needs to be updated or is it updated automatically after Fetch?

I hope you will understand me
